javascript/javascript-6-object-literals.md

```diff
@@ -7,11 +7,11 @@ About 60 minutes
 - 20 min for Check for Understanding
 
 ### Prerequisites
-- [JavaScript 1 Lesson](/javascript/javascript-1.md)
-- [JavaScript 2 Lesson](/javascript/javascript-2.md)
-- [JavaScript 3 Lesson](/javascript/javascript-3.md)
-- [JavaScript 4 Lesson](/javascript/javascript-4.md)
-- [JavaScript 5 Lesson](/javascript/javascript-5.md)
+- [JavaScript 1 Lesson](../javascript/javascript-1.md)
+- [JavaScript 2 Lesson](../javascript/javascript-2.md)
+- [JavaScript 3 Lesson](../javascript/javascript-3.md)
+- [JavaScript 4 Lesson](../javascript/javascript-4.md)
+- [JavaScript 5 Lesson](../javascript/javascript-5.md)
```

Comment:
Please leave these unchanged. We only care about links that specify the full URL. Sorry if that wasn't clear in the issue text.

Reply:
we i kept  unchanged those links ..those links are not working is it okay?

Reply:
If the link are not working then I think better to keep that changes.

Reply:
Please check CONTRIBUTING.md  where link is broken ?

Reply:
@Kundan28 it's not that I want them broken but it's illogical for a link to go up one level from the directory it's in just to go back down.

These should be `./javascript-1.md` etc

Reply:
This i changed as you specified